peps/pep-0736.rst

```diff
@@ -0,0 +1,312 @@
+PEP: 736
+Title: Syntactic sugar for named arguments
+Author: Joshua Bambrick <jbambrick@google.com>,
+        Chris Angelico <rosuav@gmail.com>
+Sponsor: Guido van Rossum <guido@python.org>
```

Comment:
I'm happy to step down, I forgot that PEP editors are also acceptable sponsors. I'm at most +0 on the proposal, but I figured it deserved to become a PEP. I'm pretty backed up, so either way you're not going to get a lot of mentoring from me, alas. :-(

Reply:
> I forgot that PEP editors are also acceptable sponsors.

There was some confusion about this on [the draft PR](https://github.com/joshuabambrick/peps/pull/1#issuecomment-1837196032). See comment from @hugovk:

>https://peps.python.org/pep-0001/#submitting-a-PEP says we need at least one of:
>* core dev co-author
>* core dev sponsor
>* PEP editor sponsor
>* SC-appointed sponsor
>Chris is a PEP editor, Guido is a PEP editor and core dev.
Right now we meet the criteria with Guido as sponsor.
We could also have Chris as sponsor instead of Guido, if you prefer

I'm happy either way. Will default to removing you as sponsor if no objections in the next few days.

> I'm pretty backed up, so either way you're not going to get a lot of mentoring from me, alas.

That's no problem